services/alertmanager/alertmanager_test.go

```diff
@@ -36,3 +47,171 @@ func TestAlertmanager(t *testing.T) {
 
 	require.NoError(t, svc.IsReady(context.Background()))
 }
+
+func TestPopulateConfig(t *testing.T) {
+	t.Run("without receivers and routes", func(t *testing.T) {
+		var cfg alertmanager.Config
+		buf, err := ioutil.ReadFile(alertmanagerBaseConfigPath)
```

Comment:
Probably it makes sense to create some fake base config, and check that we are not overriding user data.

Reply:
didn't we just decide to overwrite user data? https://github.com/percona/pmm-managed/pull/589#discussion_r547989698

Reply:
Yep, but there is possible case when user value is defined, but in settings not. So we should not remove users values, we should only override them if we have another value.

Reply:
added one field to the global config to check for overwrite